latency/src/bin/z_ping.rs

```diff
@@ -72,10 +72,10 @@ async fn single(opt: Opt, config: Config) {
         session
             .put(&reskey_ping, data)
             .congestion_control(CongestionControl::Block)
-            .await
+            .wait()
```

Comment:
Suggest to have a discussion with other developers for which is a better one?

Reply:
Based on the off-line discussions with the developers, let's add an option for this to  test the performance impact of .await and wait().